app/services/ccms/applicant_add_requestor.rb

```diff
@@ -1,5 +1,11 @@
 module CCMS
-  class CreateClientRequestor < BaseRequestor
+  class ApplicantAddRequestor < BaseRequestor
+    def initialize(applicant)
+      @applicant = applicant
+      @address = applicant.address
+      super()
+    end
```

Comment:
I'd use `attr_reader` to make these accessible within the rest of class rather than passing the variable around. Perhaps even use delegate:
```ruby
attr_reader :applicant
delegate :address, to: :applicant

def initialize(applicant)
  @applicant = applicant
  super
end
```